website/common/script/ops/buy/buyMysterySet.js

```diff
@@ -27,8 +23,8 @@ module.exports = function buyMysterySet (user, req = {}, analytics, _window) {
     throw new NotFound(i18n.t('mysterySetNotFound', req.language));
   }
 
-  if (typeof _window !== 'undefined' && !req.noConfirm && _window.confirm) { // TODO move to client
-    if (!_window.confirm(i18n.t('hourglassBuyEquipSetConfirm'))) return;
+  if (!req.noConfirm && window.confirm) { // TODO move to client
```

Comment:
shouldn't this be window && window.confirm for the node env where window is undefined?

Reply:
thats what I tried too, but:
`if (window && window.confirm) ` results in
`2018-01-28T16:07:20.949Z - error: ReferenceError: window is not defined`

Reply:
try 

```
if (typeof window !== 'undefined') {
    window.confirm...
}
```

Reply:
I thought about it a second after I wrote my message haha, but now I think this is obselete, because every `hourglass` purchase has already a `purchase for 1 hourglass` confirmation on the client before even this code is called / executed, I think it can be removed completely

Reply:
ah okay!